ibis/omniscidb/tests/test_client.py

```diff
@@ -215,7 +194,7 @@ def test_read_csv(con, temp_table, filename):
     table.read_csv(filename, header=False, quotechar='"', delimiter=",")
 
     df_read_csv = table.execute()
-    df_expected = db.table("functional_alltypes").execute()
+    df_expected = alltypes.execute()
```

Comment:
@dchigarev I think you can use `df_alltypes` instead (https://github.com/ibis-project/ibis/blob/master/ibis/omniscidb/tests/conftest.py#L122)
so you don't need to execute it again.

Reply:
@xmnlab done